index.html

```diff
@@ -0,0 +1,56 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Zuhra Hashimi</title>
+</head>
+<body>
+    <header>
+        <nav class="topnav">
+            <a href="#" class="logo">Zuhra Hashimi</a>
+            <div class="page_header">
+                <img class="img" src="images/Icon - Menu.png" alt="">
+            </div>
```

Comment:
[OPTIONAL] It'd be better to wrap the hamburger in an `<a>` tag, since later on you'll add a navigation menu here that'd open when you click this hamburger 🤗